person.rb

```diff
@@ -55,6 +57,14 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
+
+  def add_rentals(rentals)
+    @rentals.concat(rentals)
+  end
+
```

Comment:
- [ ] We only need one method. Kindly remove the second one.

-  [ ] For the first one we need to refactor it to so that given a book and date, a person can create a rental.

  ```
  def add_rental(date, book)
    Rental.new(date, book, self)
  end
  ```